src/ck-core/ckrdma.C

```diff
@@ -474,7 +475,7 @@ CkNcpyMode findTransferMode(int srcPe, int destPe) {
   if(CmiNodeOf(srcPe)==CmiNodeOf(destPe))
     return CkNcpyMode::MEMCPY;
 #if CMK_USE_CMA
-  else if(CmiDoesCMAWork() && CmiPeOnSamePhysicalNode(srcPe, destPe))
+  else if(useCMAForZC && CmiDoesCMAWork() && CmiPeOnSamePhysicalNode(srcPe, destPe))
```

Comment:
It seems potentially nicer to rename CmiDoesCMAWork to CmiIsCMAEnabled() or something and to check useCMAForZC inside of it

Reply:
`CmiDoesCMAWork` is implemented in the LRTS layer and returns `true` if the machine has support for CMA (after CMA Init). 

I could add a wrapper function `CmiUseCMAForZC` that returns `useCMAForZC && CmiDoesCMAWork`.

Reply:
I forgot this only disables CMA for ZC, not for everything, so it's not needed